test/unit/misc.spec.js

```diff
@@ -0,0 +1,128 @@
+'use strict';
+const assert = require('assert');
+const misc = require('../../src/helpers/misc');
+
+describe('Misc', () => {
+  describe('#getTopicsByWildcard()', () => {
+    const mockedRegistry = [{
+      type: 'services',
+      name: 'servicesIpsum'
+    },
+      {
```

Comment:
please format spacing and will be ZAJEBIŚCIE! :D  in another payloads below the same

personally prefer formatting objects in array that:
```
const mockedRegistry = [
  {
    type: 'services',
    name: 'servicesIpsum'
  },
  {
    type: 'services',
    name: 'servicesLorem'
  }
];
```